elements/pl-dataframe/pl-dataframe.css

```diff
@@ -17,6 +17,7 @@
     box-sizing: border-box;
     color: black;
     vertical-align: bottom;
+    text-align: right;
```

Comment:
IMO the `thead th` names should be padded like the `td` cells below. Otherwise there's still enough misalignment to be confusing IMO.

Reply:
Is there a way to share that padding in this CSS somehow? I'm no expert unfortunately.

Reply:
Maybe you could use CSS variables or just specify the padding for several things at once, like the `border: none` is written below.

Reply:
Updated the CSS:

![image](https://user-images.githubusercontent.com/1345068/216680116-c388db03-3b07-43e1-a83f-862390381d0d.png)

Reply:
Is there a reason we're using right-aligned columns? It looks kinda weird for text. Is this because we don't have the ability to differentiate between numerical and text columns in the markup?

Reply:
Now it looks properly aligned 🚀 The css could be refactored, but not really urgent.

Reply:
Oops, nathan's comment hadn't loaded for me. I'll unresolve this chain.

Reply:
Yeah, not sure if there's a way to distinguish between them using the styler. The right alignment was used because that was the styling before, but could definitely be changed if there's a better way.

Reply:
That's how pandas chooses aligns everything (both in the REPL and HTML), so pandas users will be used to everything being right-aligned. I'm not sure about other libraries though. Personally I think the consistency looks better.

Reply:
I think displaying things in a way people expect is probably the way to go then. If Pandas veterans expect it to be right aligned, we should probably stick with that.